src/atomicdex/pages/qt.wallet.page.cpp

```diff
@@ -163,6 +163,7 @@ namespace atomic_dex
             obj["blocks_left"]       = static_cast<qint64>(tx_state.blocks_left);
             obj["transactions_left"] = static_cast<qint64>(tx_state.transactions_left);
             obj["current_block"]     = static_cast<qint64>(tx_state.current_block);
+            obj["is_test_coin"]      = coin_info.ticker == "RICK" || coin_info.ticker == "MORTY";
```

Comment:
`is_smartchain_test_coin` seems more reasonable here

We also have other test-coin such as `qrc-20 (test)` but we cannot have faucet on this one